doc/sources/installation/installation-windows.rst

```diff
@@ -230,6 +239,36 @@ kivy with git rather than a wheel there are some additional steps:
      export USE_GSTREAMER=1
 
    These variables must be set everytime you recompile kivy.
+   
+   **MSVC:**
```

Comment:
No, the code was already using sys.executable, which doesn't work, see note above. It's checking the ```%INCLUDE %``` that actually does anything here; which is itself a shortcut to setting the SDL header location manually, which bypasses the entire SDL preparation logic (and is therefore more reliable, which is why I doc'd it as a note).